<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qfm1000.qrc">
    <normaloff>:/icons/application</normaloff>:/icons/application</iconset>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileNew"/>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileClose"/>
    <addaction name="separator"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuProgrammer">
    <property name="title">
     <string>Programmer</string>
    </property>
    <addaction name="actionProgrammerRead"/>
    <addaction name="actionProgrammerWrite"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConfiguration"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProgrammer"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileNew"/>
   <addaction name="actionFileOpen"/>
   <addaction name="actionFileClose"/>
   <addaction name="separator"/>
   <addaction name="actionFileSave"/>
   <addaction name="actionFileSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionProgrammerRead"/>
   <addaction name="actionProgrammerWrite"/>
   <addaction name="separator"/>
   <addaction name="actionConfiguration"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/application/exit</normaloff>:/action/application/exit</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionFileNew">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/document/new</normaloff>:/action/document/new</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/document/open</normaloff>:/action/document/open</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/document/close</normaloff>:/action/document/close</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/document/save</normaloff>:/action/document/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/document/saveas</normaloff>:/action/document/saveas</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/edit/cut</normaloff>:/action/edit/cut</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/edit/copy</normaloff>:/action/edit/copy</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/edit/paste</normaloff>:/action/edit/paste</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/edit/delete</normaloff>:/action/edit/delete</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionProgrammerRead">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/eeprom/read</normaloff>:/action/eeprom/read</iconset>
   </property>
   <property name="text">
    <string>&amp;Read</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionProgrammerWrite">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/eeprom/write</normaloff>:/action/eeprom/write</iconset>
   </property>
   <property name="text">
    <string>&amp;Write</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/application/configure</normaloff>:/action/application/configure</iconset>
   </property>
   <property name="text">
    <string>&amp;Configuration</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/action/application/about</normaloff>:/action/application/about</iconset>
   </property>
   <property name="text">
    <string>A&amp;bout...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="icon">
    <iconset resource="../qfm1000.qrc">
     <normaloff>:/images/licenses/gpl-v3.svg</normaloff>:/images/licenses/gpl-v3.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;License</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qfm1000.qrc"/>
 </resources>
 <connections/>
</ui>
